pkg/diff/sql_generator.go

```diff
@@ -1167,18 +1167,21 @@ func (rsg *renameConflictingIndexSQLVertexGenerator) Add(index schema.Index) ([]
 }
 
 func (rsg *renameConflictingIndexSQLVertexGenerator) generateNonConflictingName(index schema.Index) (string, error) {
-	uuid, err := uuid.NewRandom()
+	uuid, err := pgidentifier.RandomUUID()
 	if err != nil {
-		return "", fmt.Errorf("generating UUID: %w", err)
+		return "", fmt.Errorf("generating RandomUUID: %w", err)
 	}
 
-	newNameSuffix := fmt.Sprintf("_%s", uuid.String())
+	prefix := fmt.Sprintf("%sidx_", tmpObjNamePrefix)
```

Comment:
Switch to use the canonical tmpObjectNamePrefix `pgschemadiff_tmp` and also the new base64 encoded UUID